coilsnake/util/common/helper.py

```diff
@@ -54,3 +54,8 @@ def lower_if_str(x):
 
 def grouped(iterable, n):
     return izip(*[iter(iterable)]*n)
+
```

Comment:
What happened here was Lyrositor used a function that's in a different version of Python than the one we're using, so I had to manually add the function.